src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-list/rbd-list.component.ts

```diff
@@ -376,6 +384,26 @@ export class RbdListComponent extends ListWithDetails implements OnInit {
     });
   }
 
+  resyncRbdModal() {
+    const poolName = this.selection.first().pool_name;
+    const namespace = this.selection.first().namespace;
+    const imageName = this.selection.first().name;
+    const imageSpec = new ImageSpec(poolName, namespace, imageName);
+
+    this.modalRef = this.modalService.show(CriticalConfirmationModalComponent, {
+      itemDescription: 'RBD',
+      itemNames: [imageSpec],
+      actionDescription: 'resync',
+      submitActionObservable: () =>
+        this.taskWrapper.wrapTaskAroundCall({
+          task: new FinishedTask('rbd/edit', {
+            image_spec: imageSpec.toString()
+          }),
+          call: this.rbdService.update(imageSpec, { resync: true })
+        })
+    });
+  }
```

Comment:
Same here. Any way to display some brief description of this or point users [to the docs](https://docs.ceph.com/en/quincy/rbd/rbd-mirroring/#force-image-resync)?

Reply:
This seems a separate task which could be made a bit more generic. Hence creating a [tracker](https://tracker.ceph.com/issues/56027) for this to be implemented in a different PR.

I will list some of my thoughts on this below:

Tool-tips (added adjacent to action name) could be of 2 types, one which appears on _hovering_ or _clicking_ the element.

**Hovering**: To click a link in a tool-tip you would need to move your mouse away from the element (which will hide the top-tip) and on the tool-tip (not visible anymore). This could be a bit tricky, maybe adding a delay fade out animation might work?.  

**Clicking**: This could work only when the tableAction is disabled. As, if it is enabled then while clicking on the table action might misfire a top-tip pop up. which needs to be closed by again clicking at the info-icon. If this is not an issue this could be a good option as now we can easily add and click links in tool-tips. Something similar already exists in the dashboard: `InfoGroupComponent`.

But if we want to educate the user about a specific feature then we could have a full-blown dialog-box/modal pop-up on-clicking the info-icon, to get more flexibility on the type of content we want to show.

I think this `InfoCompoment` should be more generic as it can be used in various places.

Reply:
@Sarthak0702 agree with you! According to Patternfly, the first type (hovering) are the tooltips themselves, while the other type (clicking) they [call them popovers](https://www.patternfly.org/v4/components/popover/design-guidelines/#tooltips-versus-popovers).

Pros of tooltips is that they provide immediate documentation/context and work great with accessibility (because they're defined in the element they describe). However, for popovers to work with screen readers, you must use ARIA labels to link elements (describes/described-by, since you need to use 2 unrelated elements). @nSedrickm knows much more about this than I.

I think the tooltip is great for a short snippet of documentation, while popovers might work great for linked resources.

Reply:
Ahh correct, popovers. I have seen this at various places in the dashboard. Maybe we can decouple it and make a generic component for tooptip and popover.

Pop-Over
![image](https://user-images.githubusercontent.com/48375025/173569760-bb09ccd8-1aa8-40f9-8c05-c0e305e1134b.png)

Tool-Tip
![image](https://user-images.githubusercontent.com/48375025/173570224-65c10f64-42c9-4db7-8bd9-895fdef6abdd.png)